addons/persistent-thumb/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Always show thumbnail",
+  "description": "Always shows thumbnails under the big green flag before projects are started.",
```

Comment:
```suggestion
  "description": "Fixes a bug in the Scratch project player where thumbnails don't always load before a project is started.",
```

Reply:
It isn't a bug. What you see before the project starts isn't a thumbnail, it's the state of the project when it was saved. That's why it's blank for pen projects - Scratch 3 doesn't save the pen layer.

Reply:
We can remove part about bug, but I think this part `Scratch project player where thumbnails don't always load ` should stay.

Reply:
When I was testing this addon I didnt undetstood how it works, until mxmou told about pen extension,  so maybe we can add info about it? Maybe
`This addon always shows the project thumbnail before a project is started (even if project uses pen extension).`